src/sas/qtgui/Perspectives/Corfunc/media/corfunc-theory.rst

```diff
@@ -294,7 +294,7 @@ which by the relationship between complex trigonometric and hyperbolic functions
 The leading :math:`2` will cancel the leading :math:`1/2` and the value of :math:`I(q)` can be seen to be
 
 .. math::
-    I(q) = \int_{\vec{r}\in\mathbb{R}^3} \gamma(\vec{r}) \frac{\sin qr}{qr} d\vec{r}^3
+    I(q) = \int_{r \in \mathbb{R}^3}^{\infty} \gamma(\vec{r}) \frac{\sin qr}{qr} d \vec{r}
```

Comment:
There should be a 0 limit as well as infinity over the integral, i.e., I(q) = \int_{r \in \mathbb{R}^3}_{0}^{\infty} \gamma(\vec{r}) \frac{\sin qr}{qr} d \vec{r}